vladiate/test/test_validators.py

```diff
@@ -85,6 +85,7 @@ def test_set_validator_works(field_set, field):
     ([], 'bar'),
     (['foo'], 'bar'),
     (['foo', 'bar'], 'baz'),
+    ([str(x) for x in range(200)], "notanumber"),
```

Comment:
Would you be able to add some actual tests for `stringify_set` instead of this?

Reply:
I just found out that `stringify_set` may not be deterministic since the output is dependent on the order of elements that are generated by the set iterator.

For Python 2.7.12:
```
>>> {'A', 'B', 'C', 'D', 'E', 'F'}
set(['A', 'C', 'B', 'E', 'D', 'F'])
```

For Python 3.5.2: It can be different every time the python interpreter is started!